app/Signup/page.tsx

```diff
@@ -1,9 +1,146 @@
-import styles from "./page.module.css"
+'use client';
+import styles from "./page.module.css";
+import { useState} from "react";
 
 export default function Page() {
+
+    const [user, setUser] = useState({
+        firstName: "",
+        lastName: "",
+        userName: "",
+        houseName: "",
+        userEmail: "",
+        userPassword: "" 
+    });
+    const [password, setPassword] = useState("");
+    const [rePassword, setRePassword] = useState("");
+    const [houseError, setHouseError] = useState(false);
+    const [emailError, setEmailError] = useState(false);
+    const [userPasswordError, setUserPasswordError] = useState(false);
+    const [userRePasswordError, setUserRePasswordError] = useState(false);
+    const hasError = houseError || emailError || userPasswordError || userRePasswordError;
+
+    const validateHouseName = (e) => {
+        const alphaNumericCheck = /^[a-z\d\-_\s]+$/i;
+        if ( !alphaNumericCheck.test(e.target.value)) {
+            setHouseError(true);
+        }
+        else
+        setHouseError(false);
```

Comment:
Remove else statement. At the end of the function you don't have to put a "return" keyword. If one isn't provided at the end it'll implicitly return null.